user_modules/shared_testcases/testCasesForPolicyTable.lua

```diff
@@ -927,9 +926,12 @@ function testCasesForPolicyTable:flow_SUCCEESS_EXTERNAL_PROPRIETARY(self, app_id
   EXPECT_HMIRESPONSE(RequestId_GetUrls,{result = {code = 0, method = "SDL.GetURLS", urls = endpoints} } )
   :Do(function(_,_)
     self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",
-    { requestType = "PROPRIETARY", fileName = ptu_file_name})
+    { requestType = "PROPRIETARY", fileName = SystemFilesPath .. pts_file_name})
```

Comment:
@istoimenova, It was clarified with BA that only fileName (without path) has to be provided
So please check update in 1af6fd0